egress/types.go

```diff
@@ -35,12 +36,20 @@ type EncodedOutput interface {
 	GetFileOutputs() []*livekit.EncodedFileOutput
 	GetStreamOutputs() []*livekit.StreamOutput
 	GetSegmentOutputs() []*livekit.SegmentedFileOutput
+	ImageOutput
 }
 
 type EncodedOutputDeprecated interface {
 	GetFile() *livekit.EncodedFileOutput
 	GetStream() *livekit.StreamOutput
 	GetSegments() *livekit.SegmentedFileOutput
+	GetFileOutputs() []*livekit.EncodedFileOutput
+	GetStreamOutputs() []*livekit.StreamOutput
+	GetSegmentOutputs() []*livekit.SegmentedFileOutput
+}
+
+type ImageOutput interface {
+	GetImageOutputs() []*livekit.ImageOutput
```

Comment:
Is there a reason for this to be separate?

Reply:
nvm, we'll now have 2 kinds of deprecated requests - the oldest having only GetX, one having GetXOutputs without Images, and the newest having GetXOutputs including images